src/views/transaction/transaction.view.jsx

```diff
@@ -41,7 +41,7 @@ function Transaction ({
       case TxType.Withdraw:
       case TxType.Exit:
       case TxType.ForceExit: {
-        return 'Withdraw'
+        return 'Withdraw Step 1/2 Completed'
```

Comment:
@chamian do you agree?

Reply:
Latest changed added as per Chami's request.

Reply:
<img width="920" alt="Screenshot 2021-05-25 at 12 02 35" src="https://user-images.githubusercontent.com/2448210/119509350-dc016d80-bd70-11eb-903e-07613a54d532.png">